app/javascript/packs/search_tasks.js

```diff
@@ -1,5 +1,5 @@
 
-$(document).ready(() => {
+$(document).on("turbolinks:load", () => {
```

Comment:
To make the searching work when coming back to tasks page after clicking `back` button